curriculum/challenges/english/25-front-end-development/review-react-basics/67487e141bb6a7140a352e12.md

```diff
@@ -0,0 +1,253 @@
+---
+id: 67487e141bb6a7140a352e12
+title: React Basics Review
+challengeType: 24
+dashedName: review-react-basics
+---
+
+# --description--
+
+## JavaScript Libraries and Frameworks
+
+- JavaScript libraries and frameworks offer quick solutions to common problems and speed up development by providing pre-built code.
+- Libraries are generally more focused on providing solutions to specific tasks, such as manipulating the DOM, handling events, or managing AJAX requests. 
+- An example of a JavaScript library is jQuery.
+- Frameworks, on the other hand, provide a more defined structure for building applications. They often come with a set of rules and conventions that developers need to follow.
+- Examples of frameworks include Angular and React.
+- **Single page applications**(SPAs) are web applications that load a single HTML page and dynamically update that page as the user interacts with the application without reloading the entire page.
```

Comment:
We never officially discussed this but there is a difference in formatting here between the different review pages.
Since we will have different team members in 2025 working on these pages there should be a consensus.

Do we want just bullet points?
Or the way it was originally setup? 

```md
- **topic**: placeholder
```